firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Part.kt

```diff
@@ -57,11 +57,11 @@ public class ImagePart(public val image: Bitmap) : Part
 public class InlineDataPart(public val inlineData: ByteArray, public val mimeType: String) : Part {
 
   @Serializable
-  internal data class Internal(@SerialName("inline_data") val inlineData: InlineData) :
+  internal data class Internal(@SerialName("inlineData") val inlineData: InlineData) :
```

Comment:
Why were these changed?

Reply:
somehow bidi server doesnt recognize inline_data but recognizes inlineData

Reply:
That's concerning but I suppose fine

Reply:
if name is the same, no need for serialname